pkg/reconciler/v1alpha1/service/resources/route.go

```diff
@@ -26,6 +26,9 @@ import (
 	"github.com/knative/serving/pkg/reconciler/v1alpha1/service/resources/names"
 )
 
+// See #2819.
+const releaseLastRevisionKeyword = "@latestRevision"
```

Comment:
Make public and reuse the same constant?

Reply:
duh already done :)